bsp/src/org/jetbrains/bsp/project/importing/projectImport.scala

```diff
@@ -203,14 +212,18 @@ class BspProjectImportProvider(builder: BspProjectImportBuilder)
 
   override def canImport(fileOrDirectory: VirtualFile, project: Project): Boolean =
     BspProjectOpenProcessor.canOpenProject(fileOrDirectory) ||
-      SbtProjectImportProvider.canImport(fileOrDirectory)
+      SbtProjectImportProvider.canImport(fileOrDirectory) ||
+      FastpassProjectImportProvider.canImport(fileOrDirectory)
 
   override def createSteps(context: WizardContext): Array[ModuleWizardStep] = {
     builder.reset()
     builder.autoConfigure(context.getProjectDirectory.toFile)
+    builder.setFileToImport(context.getProjectDirectory.toString)
 
-    Array(
-      new BspSetupConfigStep(context, builder),
+    // Calling super.createSteps here is essential to ensure that
+    // the `NewProjectUtil.doCreate` is called *after* resolveProjectInfo(isPreview=true)
+    super.createSteps(context) ++ Array(
```

Comment:
This causes an additional dialog on import, which isn't needed most of the time. Can we avoid it?

Reply:
Ok, managed to do it without the extra step, just commited. Nevertheless, I'm really not sure if it's safe to skip `super.createSteps` here